### PR TITLE
Feature/tickets/dm 26111

### PR DESCRIPTION
**Pull request by andrewbheyer**

### CODE REVIEW

File: Jenkinsfile.conda [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+        OSPL_HOME="/opt/OpenSpliceDDS/V6.9.0/HDE/x86_64.linux"
+    }
+    options {
+        buildDiscarder
+            (logRotator (
+                artifactDaysToKeepStr: '',
+                artifactNumToKeepStr: '',
+                daysToKeepStr: '14',
+                numToKeepStr: '10'
+            ) ),
```
Comment (couger01) [inline Jenkinsfile.conda:23]:
```suggestion
        buildDiscarder(
            logRotator(
                artifactDaysToKeepStr: '',
                artifactNumToKeepStr: '',
                daysToKeepStr: '14',
                numToKeepStr: '10'
            ))
```

File: Jenkinsfile.conda [hunk 2]
```diff
@@ -0,0 +1,101 @@
+pipeline {
+    agent{ 
+        docker {
+            image 'lsstts/conda_package_builder:latest'
+            alwaysPull 'true'
+            args "--env {csc_config_repo}_DIR=/home/saluser/{csc_config_repo} --env LSST_DDS_DOMAIN=citest -u root --entrypoint=''"
```
Comment (couger01) [inline Jenkinsfile.conda:6]: Replace {csc_config_repo} with actual configuration repo.

File: Jenkinsfile.conda [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+            # csc_config_repo should be changed to the CSC's configuration repo name i.e. ts_config_attcs for the athexapod.
```
Comment (couger01) [inline Jenkinsfile.conda:7]:
```suggestion
            // csc_config_repo should be changed to the CSC's configuration repo name i.e. ts_config_attcs for the athexapod.
```

File: Jenkinsfile.conda [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+            # These could be changed to environment variables but that has not been tested yet, so does not make it into the form of the template.
```
Comment (couger01) [inline Jenkinsfile.conda:8]:
```suggestion
            // These could be changed to environment variables but that has not been tested yet, so does not make it into the form of the template.
```

File: Jenkinsfile.conda [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+        }
+    }
+    environment {
+        package_name = "ts_fiberspectrograph"
+        # name of the csc package i.e. ts_athexapod
```
Comment (couger01) [inline Jenkinsfile.conda:13]:
```suggestion
        // name of the csc package i.e. ts_athexapod
```

**Comment by timj** [comment]
@andrewbheyer there's a bogus merge commit in the history. Please rebase before merging.

**Pull request merged**